build.gradle

```diff
@@ -2,36 +2,38 @@ buildscript {
   ext {
     versions = [
       // Kotlin
-      KOTLIN: '1.3.72',
+      KOTLIN: '1.7.10',
       // Gradle
-      GRADLE: '4.2.0',
+      GRADLE: '7.2.1',
       // gRPC
-      COMMON_PROTOS: '1.18.1',
-      GRPC  : '1.31.1',
+      COMMON_PROTOS: '2.9.1',
+      GRPC  : '1.47.0',
       JAVAX_ANNOTATION: '1.3.2',
-      PROTOC: '3.11.4',
+      PROTOC: '3.21.2',
+      PROTO_JAVALITE: '3.21.2',
       PROTOC_JAVALITE: '3.0.0',
-      PROTOBUF_GRADLE: '0.8.10',
+      PROTOBUF_GRADLE: '0.8.19',
       // Google
-      GOOGLE_PLAY_SERVICES: '17.2.1',
+      GOOGLE_PLAY_SERVICES: '18.1.0',
       // Test
-      ANDROIDX_TEST_ESPRESSO: '3.2.0',
-      ANDROIDX_TEST: '1.1.1',
-      ANDROIDX_TEST_EXT: '1.1.1',
-      JUNIT: '4.13',
-      KLUENT: '1.60',
-      MOCKITO_ANDROID: '3.3.3',
-      MOCKITO_KOTLIN: '1.6.0',
-      ROBOLECTRIC: '4.5.1',
+      ANDROIDX_TEST_ESPRESSO: '3.4.0',
+      ANDROIDX_TEST: '1.4.0',
+      ANDROIDX_TEST_ORCHESTRATOR: '1.4.1',
+      ANDROIDX_TEST_EXT: '1.1.3',
+      JUNIT: '4.13.2',
+      KLUENT: '1.68',
+      MOCKITO_ANDROID: '4.6.1',
+      MOCKITO_KOTLIN: '4.0.0',
+      ROBOLECTRIC: '4.8.1',
       // Publish
-      DOKKA: '0.10.1',
-      NEXUS_STAGING: '0.22.0',
+      DOKKA: '1.7.10',
+      NEXUS_STAGING: '0.30.0',
```

Comment:
https://github.com/Codearte/gradle-nexus-staging-plugin

> IMPORTANT. To make releasing to Maven Central even easier, I and Marc Phillip (the author of nexus-publish-plugin) combined forces to create a next generation, unified, 2-in-1 plugin - gradle-nexus-publish-plugin. It is a recommended solution, as our development effort will be put in that new plugin. See my blog post and the official migration guide.

I will migrate to gradle-nexus-publish-plugin in the following PR.